utility/src/main/java/org/oppia/android/util/parser/CustomBulletSpan.kt

```diff
@@ -22,18 +22,22 @@ class CustomBulletSpan(context: Context) : LeadingMarginSpan {
   private var gapWidth: Int = 0
   private var yOffset: Int = 0
   private var bulletLeadingMargin: Int = 0
+  private var textLeadingMargin: Int = 0
```

Comment:
Here & above:  please make these vals (we should avoid using vars and prefer immutability where we can since that results in safer & easier-to-maintain code).

Also, once you convert these to vals you can remove all of their initializations since they're being inited in the ``init`` block below.

Reply:
Done